packages/website/src/styles.css

```diff
@@ -44,7 +44,7 @@ body {
   @apply !outline-none;
 }
 ::selection {
-  @apply bg-primary text-black;
+  @apply bg-[#BE132D] text-black;
```

Comment:
I don't think this is the correct fix.
Please follow it up in your PR. @sandren